gradle.properties

```diff
@@ -7,6 +7,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 # MP
 kotlin.mpp.androidSourceSetLayoutVersion=2
-android.defaults.buildfeatures.buildconfig=true
-android.nonTransitiveRClass=false
-android.nonFinalResIds=false
\ No newline at end of file
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
Teď na to koukám a nevšimnul jsem si toho. Nechtělo by se ti ještě udělat, že BuildConfig by měl jen androidMain? Ostatní moduly by ho měli implicitně vypnutů, jen ten androidMain by ho měl zaplý

Reply:
Môžem ho vypnúť pre modul dediaci z AndroidApplication (čo ho mal teraz zapnutý), ale neviem, ako ho mať zapnutý iba pre jeden source (androidMain) a vypnúť ho pre ostatné v rámci jedného modulu (shared) 🤔

Reply:
Opravené v rámci Convention plugins CR